www/js/app.js

```diff
@@ -166,7 +165,24 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     
     // Unique identifier of the article expected to be displayed
     var expectedArticleURLToBeDisplayed = "";
-    
+ 
+    // define and store dark preference for matchMedia
+    var darkPreference = window.matchMedia('(prefers-color-scheme:dark)');
+    // if 'prefers-color-scheme' is not supported in the browser
+    if (window.matchMedia('(prefers-color-scheme)').media === 'not all') {
+        var optionsToBeRemoved = document.querySelectorAll('.autoNotAvailable');
+        for (var i = 0; i < optionsToBeRemoved.length; i++) {
+            optionsToBeRemoved[i].remove();
```

Comment:
Unfortunately `ChildNode.remove()` isn't available in IE11, so this code won't work. See https://caniuse.com/?search=remove . To test in IE11, download Microsoft Edge, and run it in "Internet Explorer mode". Opera isn't a good analogue for IE11 because it is based on Chromium.

Reply:
Then I guess I should try `removeChild` method